src/middlewared/middlewared/plugins/smb.py

```diff
@@ -459,6 +459,21 @@ def check_codec(encoding):
             if enableacct.returncode != 0:
                 raise CallError(f'Failed to enable {username}: {enableacct.stderr.decode()}')
 
+    @private
+    def wsdd_cmd(self, cmd=None):
```

Comment:
Do you see this being used outside service plugin?

Reply:
At this point I don't see it being used outside of the SMB plugin. It's fairly tightly coupled with smbd. wsdd won't actually start unless "enable web service discovery" is set in the smb4.conf file.

Reply:
I meant service plugin, as in, why this method is not in service plugin if all this handles in service cmd?